<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceEvolverGUI</class>
 <widget class="QMainWindow" name="SurfaceEvolverGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>766</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777213</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SurfaceEvolverGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QToolButton" name="bgColorButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="bgColorLabel">
          <property name="text">
           <string>Background color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Mesh render mode:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBoxVertices">
             <property name="text">
              <string>Vertices</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxWireframe">
             <property name="text">
              <string>Wireframe</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxSurface">
             <property name="text">
              <string>Surface</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="ActionOpenFile">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionDoSomething">
   <property name="text">
    <string>DoSomething</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bgColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SurfaceEvolverGUI</receiver>
   <slot>ActionRendererBackgroundColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxVertices</sender>
   <signal>clicked(bool)</signal>
   <receiver>SurfaceEvolverGUI</receiver>
   <slot>ActionRenderVertices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxWireframe</sender>
   <signal>clicked(bool)</signal>
   <receiver>SurfaceEvolverGUI</receiver>
   <slot>ActionRenderWireframe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSurface</sender>
   <signal>clicked(bool)</signal>
   <receiver>SurfaceEvolverGUI</receiver>
   <slot>ActionRenderSurface()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ActionRendererBackgroundColor()</slot>
  <slot>ActionRenderVertices()</slot>
  <slot>ActionRenderWireframe()</slot>
  <slot>ActionRenderSurface()</slot>
 </slots>
</ui>
